tasks/connectors/edgescan/lib/edgescan_vulnerability.rb

```diff
@@ -50,11 +53,21 @@ def to_kenna_finding
         end
 
         def to_corresponding_kenna_asset
-          { "external_id" => external_asset_id, "tags" => asset.tags }
+          base = { "external_id" => external_asset_id, "tags" => asset.tags }
+          if ip_address?
+            base["ip"] = location
+          elsif url?
+            base["url"] = location
+          else
```

Comment:
Is there a way to retrieve _both_ the IP address and the hostname associated with a particular system?

i.e.

```
{
  "ip_address": "10.15.1.32",
  "hostname": example.pizza,
  "tags": ["YOLO"],
}
```

Reply:
Not when creating an asset from the Vulnerability no. Since this is used when no Host is available we don't have the IP and hostname available.